Units/parser-pascal.r/simple-pascal.d/args.ctags

```diff
@@ -0,0 +1 @@
+--fields=+tS
```

Comment:
Could you add
```
--sort=no
```
?

So reviewing the test case becomes easier a bit.

Reply:
Done (I will squash the commits before merge).

Reply:
I see. Thank you.